.codeclimate.yml

```diff
@@ -0,0 +1,9 @@
+engines:
+  eslint:
+    enabled: true
+  csslint:
+    enabled: true
+    
+ratings:
+  paths:
+    - "DOL.WHD.Section14c.Web/src/**"
```

Comment:
In our testing CodeClimate was picking up our eslint config and is scanning the front end files: https://codeclimate.com/github/AppliedIS/dol-whd-14c/code?page=3

The errors on ln 744 of the log are when Karma runs the preprocessor for the Istanbul instrumentation within the webpack.  The main eslint does succeed (Ln: 160), the local `npm start` .  We did some initial research and were unable to resolve them, but will follow up.